certificate-authority/config.yaml

```diff
@@ -68,6 +69,28 @@ clients:
         timeout: 1m0s
         throttleTime: 500ms
         documentLimit: 1000
+    cqlDB:
```

Comment:
configuration of the database can differ even though their query language is the same. Replication factor, numConnections, and other properties might or might not exist or use different names. Therefore I would propose to use directly scyllaDB as a key.